### PR TITLE
add GeoSeries for arctern

### PR DESCRIPTION
**Pull request by shengjh**
This work include 2 items:

1. Update python wrapper_func api to support more parameter types #617 
2. Add a GeoSeries class for arctern, and some unittese

### CODE REVIEW

**Comment by shengjh** [comment]
@nezha2017 @czs007 please review the code.
@become-nice @yxm1536 @Emma-Song  please review the docstring in `geoseries.py`.

File: python/tests/geo/geo_test.py [hunk 1]
```diff
@@ -599,4 +599,4 @@ def test_ST_Envelope_Aggr():
     p2 = "POLYGON ((5 1,7 1,7 2,5 2,5 1))"
     data = pandas.Series([p1, p2])
     rst = arctern.ST_AsText(arctern.ST_Envelope_Aggr(arctern.ST_GeomFromText(data)))
-    assert rst[0] == "POLYGON ((0 0,0 4,7 4,7 0,0 0))"
+    assert rst[0] == "POLYGON ((0 0,0 4,7 4,7 0,0 0))"
```
Comment (czs007) [inline python/tests/geo/geo_test.py:602]: add a newline at the end of the file.

File: python/arctern/geoseries/geoseries.py [hunk 2]
```diff
@@ -0,0 +1,1041 @@
...
+
+
+def _property_geo(op, this):
+    # type: (function, GeoSeries) -> GeoSeries
+    return GeoSeries(op(this.values), index=this.index, crs=this.crs)
+
+
+def _unary_geo(op, this, *args, **kwargs):
+    # type: (function, GeoSeries, args, kwargs) -> GeoSeries
+    if 'crs' in kwargs.keys():
```
Comment (czs007) [inline python/arctern/geoseries/geoseries.py:35]: crs = kwargs.pop('crs', this.crs)

File: python/arctern/geoseries/geoseries.py [hunk 3]
```diff
@@ -0,0 +1,1041 @@
...
+        >>> from arctern import GeoSeries
+        >>> s = GeoSeries(["POINT (1 1)", None])
+        >>> s.isna()
+        0    False
+        1     True
+        dtype: bool
+        """
+        return super().notna()
+
+    def unique(self):
```
Comment (czs007) [inline python/arctern/geoseries/geoseries.py:254]: unnecessary implementation

**Pull request merged**